packages/client/src/pages/profile/changePasswordModal/ui.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { ChangePasswordModalProps } from './types';
+import { Button, Form, Input, Modal } from 'antd';
+import { useStore } from 'effector-react';
+import { setPasswordFx } from 'pages/profile/model/effects';
+import { ChangePasswordRequest } from 'shared/api/swagger';
+import { SpaceButtons } from 'pages/profile/styles';
+import { setPassword } from 'pages/profile/model';
+
+export const ChangePasswordModal: React.FC<ChangePasswordModalProps> = ({
```

Comment:
В проекте должно быть однообразие 
у нас везде просто FC
```suggestion
export const ChangePasswordModal: FC<ChangePasswordModalProps> = ({
```

Reply:
Поправлю